lib/tasks/temporary/profile_onboarding_fields.rake

```diff
@@ -0,0 +1,12 @@
+namespace :profile_onboarding_fields do
+  desc "Setup Onboarding with previously shown fields"
```

Comment:
A temporary rake tasks to keep the onboarding fields for DEV and other existing Forems

Reply:
The current fields include: 

1. Bio -> Summary

2. Where are you located? -> location
3. What is your title? -> employment_title
4. Where do you work? -> employer_name

Reply:
One issue I see here is that the labels are used both during onboarding and on the dynamically generated profile page. Currently, on DEV the wording between onboarding and the profile page isn't the same, but now it would be which may look a bit off on the profile page where most things aren't phrased as questions.

Reply:
This is a good point, and probably thee time to loop in @forem/design. Slack discussion here: https://dev-internal.slack.com/archives/C0186BE65TN/p1606986701034100 and will post the results below.

Reply:
**Result of discussion:**
>Yea, lets keep them as short labels in both places: onboarding & profile settings.
Questions were nice, but I understand how it will add a layer of complexity to everything so lets skip it for now. Short labels are still understandable enough. We can circle back when/if we decide to rethink/rebuild/refactor the Onboarding."

Will revert these changes.